src/game.rs

```diff
@@ -112,6 +113,12 @@ impl GameState {
     }
 
     pub fn play(&mut self, cnv: &mut Canvas, inp: &mut KeyReader, hs_disp: &Vec<&String>) -> u64 {
+
+        let pause_top    = "╔══════╗".to_string();
```

Comment:
I would prefer it to use `draw_strs` (see line 270 for an example usage) and then move this to the top as a const like `BORDER` (and also add a space between border and text) :

```
const PAUSE: [&'static str; DISP_HEIGHT as usize] = [
    "╔════════╗",
    "║ PAUSED ║",
    "╚════════╝"
];
```

And then call it via `cnv.draw_strs(&PAUSE.to_vec(), (7, 13), SCORE_COLOR, &Reset)`